### ISSUE TITLE
`fbrnch bugs` does not work at all

### ISSUE THREAD

**Issue by gotmax23** [juhp/hsbugzilla#18]
``` console
$ fbrnch bugs fbrnch

= fbrnch =
fbrnch: BugzillaJSONParseError "Error in $.bugs[0]: key \"see_also\" not found"
```

I suspect this has to do with the recent RH Bugzilla update.

**Comment by juhp** [comment]
Reproduced - thank you for reporting this

**Comment by juhp** [comment]
Any idea what changed in RH bugzilla? I need to debug more...

I looked over the [release notes](https://bugzilla.redhat.com/page.cgi?id=release-notes.html) too.

**Comment by juhp** [comment]
Actually it might be what is says: seems the `see_also` key does not appear in Bug's (by default anyway) anymore...
(Maybe I should relax the json parsing to prevent these kind of problems.)

**Comment by juhp** [comment]
I opened https://bugzilla.redhat.com/show_bug.cgi?id=2142477 to confirm this change.

**Comment by juhp** [comment]
It seems the Red Hat bz change is indeed intended so I will go ahead with pushing a fix and make a new release.

**Comment by juhp** [comment]
Should be fixed in bugzilla-redhat-1.0.1, which I plan to push to Fedora soon.
